### PR TITLE
NamespaceTest added

### LINKED ISSUE #1381: Test coverage missing for multiple resources
The following resources don't have any test coverage

  - [ ] Endpoints
  - [ ] PersistentVolumeClaim
  - [ ] PersistentVolumes
  - [ ] SubjectAccessReview
  - [ ] PodPreset
  - [ ] HorizontalPodAutoscaler
  - [ ] Namespace
  - [ ] ResourceQuota

### PR DESCRIPTION
**Pull request by kunal-kushwaha**
Fixes #1381
Test coverage provided for Namespace.

### CODE REVIEW

**Comment by rohanKanojia** [comment]
ok to test

File: kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/NamespaceTest.java [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+public class NamespaceTest {
+
+  @Rule
+  public KubernetesServer server = new KubernetesServer();
+
+  @Test
+  public void testList(){
```
Comment (rohanKanojia) [inline kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/NamespaceTest.java]: nit, Please keep a space between `()` and `{`

File: kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/NamespaceTest.java [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    server.expect().withPath("/api/v1/namespaces/namespace2").andReturn(200, new NamespaceBuilder().build()).once();
+
+    KubernetesClient client = server.getClient();
+    Boolean deleted = client.namespaces().withName("namespace1").delete();
+    assertTrue(deleted);
+
+    deleted = client.namespaces().withName("namespace2").delete();
+    assertTrue(deleted);
+
+    deleted = client.namespaces().withName("namespace2").delete();
```
Comment (rohanKanojia) [inline kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/NamespaceTest.java]: why are you querying server twice when you've mocked response on that link for only once?

Comment (kunal-kushwaha) [reply kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/NamespaceTest.java]: Oh, my bad

**Pull request merged**